Tests/Repository/Vcs/GitDriverTest.php

```diff
@@ -102,6 +102,67 @@ public function testPublicRepositoryWithEmptyComposer($type, $filename)
      * @param string $type
      * @param string $filename
      */
+    public function testPublicRepositoryWithSkipUpdate($type, $filename)
+    {
+        $repoUrl = 'https://github.com/fxpio/composer-asset-plugin.git';
+        $identifier = '92bebbfdcde75ef2368317830e54b605bc938123';
+        $io = $this->getMockBuilder('Composer\IO\IOInterface')->getMock();
+
+        $this->config->merge([
+            'config' => [
+                'fxp-asset' => [
+                    'git-driver' => [
+                        'skip-update' => '1 week',
+                    ],
+                ],
+                // TODO: bring code-coverage to 100%
```

Comment:
Should be removed. I just marked this, because I thought this *would* bring the coverage back to 100%.